static/js/iframe-common.js

```diff
@@ -70,6 +70,8 @@ export function generateIFrame(domain, answersExperienceFrame) {
   iframe.frameBorder = 0;
 
    // For dynamic iFrame sizing
+  iframe.style.height = '100%';
+  iframe.style.minHeight = '100%';
```

Comment:
I'm a little uncertain about this solution. There are a lot of potential problems with setting 100% height. 

Can we take a look at this with a test across a few different browsers?